public_html/wp-content/mu-plugins/blocks/source/block/index.js

```diff
@@ -0,0 +1,11 @@
+import * as organizers from './organizers';
```

Comment:
I'd actually prefer `src`, because what I see more commonplace. 
I'm curious, can `wp-scripts` could be configured to use with `source`? If so then might as well keep it this way.

Reply:
Aw man, I picked `source` because I remembered that you preferred spelling things out! 😆 

It's unclear to me right now if/how we would migrate to a pure usage of `wp-scripts` instead of our own config, but if we did, I think it would be easy to change it to `src` at that time if necessary. There are very few places in the code that need to reference that part of the path.

Reply:
>  can wp-scripts could be configured to use with source?

I think so, but it'd require a custom webpack config, which would default a lot of the purpose of using it. There may be a way to do it with simple npm script parameters or something, though.

> It's unclear to me right now if/how we would migrate to a pure usage of wp-scripts instead of our own config, but if we did, I think it would be easy to change it to src at that time if necessary. 

Good point, I withdrawal my objection :)